Sources/MixpanelInstance.swift

```diff
@@ -1017,17 +1020,34 @@ extension MixpanelInstance {
      - parameter properties: properties dictionary
      */
     public func track(event: String?, properties: Properties? = nil) {
+        track(event: event, withID: nil, properties: properties)
```

Comment:
The call site for all of these methods remains the same for backwards compatibility, and under the hood they just call the other method that uses the event id.